### PR TITLE
Fix autoscaler not working

### PR DESCRIPTION
**Pull request by eranelbaz**

### CODE REVIEW

**Comment by eranelbaz** [comment]
[Auto scaling not auto scaling](https://trello.com/c/TZ0pWXug/3813-auto-scaling-not-auto-scaling)

File: main.tf [hunk 1]
```diff
@@ -1,22 +1,33 @@
 data "aws_region" "current" {}
 
 resource "helm_release" "cluster_autoscaler" {
-  depends_on = [var.mod_dependency]
+  depends_on = [
```
Comment (yaronya) [inline main.tf:4]: Why's the newline?

**Review by yaronya — APPROVED** [review]

File: main.tf (hunk 1 above)
Comment (eranelbaz) [reply main.tf:4]: `terraform fmt`

Comment (eranelbaz) [reply main.tf]: https://github.com/env0/terraform-aws-eks-cluster-autoscaler/pull/3/commits/c430bd1c1588c36b37903c83248627c6034ab960

**Pull request merged**